crates/nu-command/src/filters/each.rs

```diff
@@ -106,7 +101,7 @@ with 'transpose' first."#
             },
             Example {
                 example: r#"[1 2 3] | each --keep-empty {|e| if $e == 2 { "found 2!"} }"#,
-                description: "Iterate over each element, keeping all results",
+                description: "Iterate over each element, keeping null results",
```

Comment:
Should we also change the flag? If the flag is `--keep-empty` now, I wonder if we should make it `--keep-nulls`

Reply:
That flag uses `is_nothing()`, which considers `PipelineData::Empty` equivalent to `Value::Nothing`. So, `--keep-empty` is technically just as accurate. (Though, while I think it's more useful for end users to think of `PipelineData::Empty` as being the same as `null`, there are some slight differences here and there that really need to be documented in more detail.)